src/java.base/share/classes/javax/crypto/JceSecurityManager.java

```diff
@@ -61,17 +58,24 @@
 
     // singleton instance
     static final JceSecurityManager INSTANCE;
+    static final StackWalker WALKER;
 
     static {
         defaultPolicy = JceSecurity.getDefaultPolicy();
         exemptPolicy = JceSecurity.getExemptPolicy();
         allPerm = CryptoAllPermission.INSTANCE;
-        INSTANCE = AccessController.doPrivileged(
-                new PrivilegedAction<>() {
-                    public JceSecurityManager run() {
-                        return new JceSecurityManager();
-                    }
-                });
+
+        @SuppressWarnings("removal")
+        JceSecurityManager dummySecurityManager = AccessController.doPrivileged(
+                (PrivilegedAction<JceSecurityManager>) JceSecurityManager::new);
+        INSTANCE = dummySecurityManager;
+
+        @SuppressWarnings("removal")
+        StackWalker dummyWalker = AccessController.doPrivileged(
+                (PrivilegedAction<StackWalker>) (() -> StackWalker.getInstance(
+                        Option.RETAIN_CLASS_REFERENCE)));
```

Comment:
An alternative is to declare a local variable of `PrivilegedAction<StackWalker>` to avoid the cast:

```
   PrivilegedAction<StackWalker> pa =
               () -> StackWalker.getInstance(Option.RETAIN_CLASS_REFERENCE);
   StackWalker dummyWalker = AccessController.doPrivileged(pa);         
```

Reply:
Reverted to the original style without the cast.